srcpkgs/libbitcoin-client/template

```diff
@@ -0,0 +1,56 @@
+# Template file for 'libbitcoin-client'
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+pkgname="libbitcoin-client"
+version=3.5.0
+revision=1
+short_desc="Bitcoin Client Query Library"
+makedepends+=" boost-devel"
+makedepends+=" icu-devel"
+makedepends+=" libbitcoin-protocol-devel"
+makedepends+=" libbitcoin-system-devel"
+makedepends+=" libpng-devel"
+makedepends+=" libsecp256k1-devel"
+makedepends+=" protobuf-devel"
+makedepends+=" qrencode-devel"
+makedepends+=" zeromq-devel"
+hostmakedepends+=" autoconf"
+hostmakedepends+=" automake"
+hostmakedepends+=" git"
+hostmakedepends+=" libtool"
+hostmakedepends+=" m4"
+hostmakedepends+=" pkg-config"
+homepage="https://github.com/libbitcoin/libbitcoin-client"
+license="AGPL-3"
+distfiles="https://codeload.github.com/libbitcoin/$pkgname/tar.gz/v$version>$pkgname-$version.tar.gz"
+checksum="bafa26647f334ecad04fc4bbef507a1954d7e0682f07bd38b90ab66dba5fe0d2"
+build_style="gnu-configure"
+configure_args+=" --with-gnu-ld"
+configure_args+=" --without-examples"
+if [[ -n "$CROSS_BUILD" ]]; then
+	configure_args+=" --host=$XBPS_CROSS_TRIPLET"
+	configure_args+=" --with-boost=$XBPS_CROSS_BASE/usr"
+	configure_args+=" --with-boost-libdir=$XBPS_CROSS_BASE/usr/lib"
+fi
+shlib_provides="libbitcoin-client.so.0"
+
+pre_configure() {
+	./autogen.sh
+}
+
+post_install() {
+	vlicense COPYING
+	vdoc README.md
+}
+
+libbitcoin-client-devel_package() {
+	short_desc+=" - development files"
+	depends="$sourcepkg>=${version}_$revision"
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+		vmove usr/lib/pkgconfig
+	}
+}
+
```

Comment:
remove this blank line and the vim line below

also apply the same corrections i pointed out here on the other templates.